templates/user/edit.html

```diff
@@ -3,21 +3,22 @@
 
 {% block content %}
 <div class='col'>
-  {{ Alert('This form does not yet function',
-    message="<p>Functionality of this form is pending more engineering work. Engineers, please remove this alert when done.</p>",
-    level='warning'
-  ) }}
+
+  {% if updated %}
+    {{ Alert('User information updated.', level='success') }}
+  {% endif %}
 
   <div class='panel'>
     <div class='panel__heading'>
       <h1>
         <div class='h2'>{{ user.first_name }} {{ user.last_name }}</div>
+        <div class='h3'>DOD ID: {{ user.dod_id }}</div>
         <div class='subtitle h3'>Edit user details</div>
       </h1>
     </div>
   </div>
 
-  {% set form_action = url_for('atst.save_user') %}
+  {% set form_action = url_for('users.user') %}
```

Comment:
Probably doesn't make any difference, but it may be clearer to use `users.update_user` here.
```suggestion
  {% set form_action = url_for('users.update_user') %}
```